practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/controller/PractitionerController.java

```diff
@@ -81,7 +82,7 @@ public PractitionerController(EntityService entityService,  SecurityContextUtil
         return ResponseEntity.status(CREATED).body(new Response(roleAssignmentId));
     }
 
-    @PreAuthorize("@authorisationService.authoriseUserRoles(#userIdentity)")
+    @PreAuthorize("@authorisationService.authUserRoles(#userIdentity)")
```

Comment:
general tidy as I wanted to make the methods less verbose to allow checkstyle to pass for line length